src/test/java/de/tum/in/www1/artemis/user/UserUtilService.java

```diff
@@ -205,21 +270,31 @@ public User createAndSaveUser(String login) {
         return userRepo.save(user);
     }
 
+    /**
+     * Creates and saves multiple Users given the amounts for each role.
+     *
+     * @param numberOfStudents    The number of students to create
+     * @param numberOfTutors      The number of tutors to create
+     * @param numberOfEditors     The number of editors to create
+     * @param numberOfInstructors The number of instructors to create
+     * @return The List of created Users
+     */
     public List<User> addUsers(int numberOfStudents, int numberOfTutors, int numberOfEditors, int numberOfInstructors) {
         return addUsers("", numberOfStudents, numberOfTutors, numberOfEditors, numberOfInstructors);
     }
 
     /**
-     * Adds the provided number of students and tutors into the user repository. Students login is a concatenation of the prefix "student" and a number counting from 1 to
-     * numberOfStudents Tutors login is a concatenation of the prefix "tutor" and a number counting from 1 to numberOfStudents Tutors are all in the "tutor" group and students in
-     * the "tumuser" group.
-     * To avoid accumulating a high number of users per course, this method also removes existing users from courses before adding new users.
+     * Creates and saves the provided number of students, tutors, editors and instructors. Also creates and saves an admin User if it does not exist.
+     * The login of the Users is a concatenation of the prefix, the role (student|tutor|editor|instructor) and a number counting from 1 to the number of Users with the
+     * corresponding role. The admin User's login is "admin". To avoid accumulating a high number of Users per Course, this method also removes existing users from Courses before
```

Comment:
Please consider rephrasing the sentence "To avoid.." in a better way to make it more understandable